src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml

```diff
@@ -22,121 +22,464 @@
 	<ResourceDictionary.ThemeDictionaries>
 
 		<ResourceDictionary x:Key="Default">
```

Comment:
Looks good! Only thing is the formatting, if you run Xaml Styler it should fix this, unless it's overridden in your XamlStyler settings but the x:Key and the ResourceKey should be on the same line for the StaticResource.

So it should look organized by property like we do in other styles like [here](https://github.com/unoplatform/Uno.Themes/blob/d5d803370cc99e11e3e895992a220b95abec477a/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L27-L50)

Reply:
Fixed